src/area-chart/chart-container.tsx

```diff
@@ -167,6 +169,7 @@ function ChartContainer<T extends AreaChartProps.DataTypes>({
             tickFormatter={yTickFormatter}
             title={yTitle}
             ariaRoleDescription={yAxisAriaRoleDescription}
+            maxWidth={maxLeftLabelsWidth}
```

Comment:
I think `width` is the width of the tick line (which spans the entire chart width when the labels correspond to the y axis —therefore the width of the chart container itself, the right side) whereas the new `maxWidth` corresponds to the width reserved for the labels (the left side). Can we make the naming more clear to avoid confusion here?

Reply:
Yes, I though about it too. We have the same problem with other cartesian chart components potentially but I can change it for the left labels and measure at least.